worker-testing-util/src/main/java/com/hpe/caf/worker/testing/QueueServicesFactory.java

```diff
@@ -58,7 +59,8 @@ public static QueueServices create(final RabbitWorkerQueueConfiguration configur
         Channel conChan = connection.createChannel();
 
         RabbitUtil.declareWorkerQueue(pubChan, configuration.getInputQueue(), configuration.getMaxPriority());
-        RabbitUtil.declareWorkerQueue(conChan, resultsQueueName, configuration.getMaxPriority());
+        if(StringUtils.isNotEmpty(resultsQueueName))
+            RabbitUtil.declareWorkerQueue(conChan, resultsQueueName, configuration.getMaxPriority());
```

Comment:
I think you should add braces around the `if` statement even though it is only one-line.